webpack/assets/javascripts/foreman_navigation.test.js

```diff
@@ -5,17 +5,10 @@ import {init} from './foreman_navigation';
 describe('initNavigation', () => {
   Object.defineProperty(window.location, 'pathname', {
     writable: true,
-    value: '/locations/3/select'
+    value: '/locations/1/select'
   });
-  $.fn.setupVerticalNavigation = jest.fn();
-  document.body.innerHTML =
-    `<div>
-         <li class="dropdown org-switcher" id="location-dropdown">
-           <a href="#" class="dropdown-toggle nav-item-iconic" data-toggle="dropdown" >
-             Loc1 <span class="caret"></span>
-           </a>
-         </li>
-       <div id="vertical-nav" class="nav-pf-vertical hover-secondary-nav-pf">    
+  const verticalMenu =
```

Comment:
should we give another name (e.g. with/out  orgs/locs?)

can you add another output that is when orgs are disabled/enabled?